test/user.test.js

```diff
@@ -401,4 +401,169 @@ describe('USER TEST SUITE', () => {
         expect(response.body.error).to.equal('Invalid token');
       });
   });
+
+  describe('User Update Validations', () => {
+    it('should return No token provided, when No token is provided',
+      async () => {
+        const response = await chai.request(server)
+          .patch('/api/v1/user');
+        expect(response.status).to.equal(400);
+        expect(response.body.error).to.equal('No token provided');
+      });
+
+
+    it('should return Invalid token, when token is Invalid',
+      async () => {
+        const response = await chai.request(server)
+          .patch('/api/v1/user')
+          .set('Authorization', 'invalidtoken');
+        expect(response.status).to.equal(401);
+        expect(response.body.error).to.equal('Invalid token');
+      });
+
+
+    it('should return User Updated Successfully,with correct parameters',
```

Comment:
`Should successfully update the user when accurate parameters are supplied`

Reply:
ok